style.css

```diff
@@ -23,13 +27,68 @@ ul {
 	list-style: none;
 }
 
-a {
+a:not(.item-link) {
 	display: block;
 	text-align: center;
-	color: #a6a2c5;
-	text-transform: uppercase;
+	color: var(--color);
+	padding: 15px;
+	text-decoration: none;
+	font-size: 25px;
+}
+
+.item-link {
 	padding: 15px;
 	text-decoration: none;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.item-icon {
+	margin-bottom: 5px;
+	transform: translateY(0px);
+	transition-duration: 450ms;
+}
+
+.item-text {
 	font-size: 25px;
+	color: var(--color);
+	font-family: sans-serif;
+	position: absolute;
+	opacity: 0;
+	transform:translateY(0) ;
+	transition-duration: 500ms;
 }
 
+.item-dot {
+	display: block;
+	width: 8px;
+	height: 8px;
+	border-radius: 50%;
+	background-color: var(--color);
+	opacity: 0;
+	transform:scale(0);
+	transition:all 500ms;
+}
+
+.item-link.active .item-icon {
+	/* Select the element with the className "item-link" and "active" and "item-icon"?*/
+	opacity: 0;
+	transform: translateY(-50px);
+}
+
+.item-link.active .item-text {
+	opacity: 1;
+	transform: translateY(-5px);
+}
+
+.item-link.active .item-dot {
+	opacity: 1;
+	transform: translateY(-5px);
+	transform:scale(1); /*1 is the original size*/
```

Comment:
In CSS you can only use a property once. When repeated, it will apply only the last one.

In this case, the `translateY` was ignored. So, you need to merge both together:

```
transform: translateY(-5px) scale(1);
```

Reply:
Oh ans apparently the order matters. The properties will be applied from right to left. Interesting...

Reply:
does the word matter for transform? I didn't know that. Can you share where you learned that?

Reply:
Here is what I found about this topic:

https://stackoverflow.com/questions/10765755/how-to-apply-multiple-transforms-in-css

Reply:
Oh, that's true!! I didn't remember that at all, nice find!